### ISSUE TITLE
Weird testing results

### ISSUE THREAD

**Issue by geekyutao** [saic-mdal/lama#113]
Dear authors,
  Thank you for the excellent repo! I replaced the residual blocks in pix2pix (a very simple model) with LaMa FFC residual blocks. During training, the inpainting results are rather good, while during testing, the inpainting results are weird (see the follows). It seems that It inpaints a small part of masked regions, leaving a large area uncorrectly inpainted/non-inpainted. Do you meet this before? Note that I set model.eval() when evaluating.
  Hope you can give some advices. Thank you!
![image](https://user-images.githubusercontent.com/30614660/165704522-3855dbbc-3d01-4c9b-a9a0-7fe2246ef53a.png)

**Comment by geekyutao** [comment]
BTW, the input is only masked image. It wasn't concatenated with mask.

**Comment by windj007** [comment]
Could you please provide more information regarding training? What data do you train on? How long have you trained the model? What are the settings of FFC blocks? What is the width of training masks?

It seems that the generator can fill small holes, but not the bigger ones... that hints that there might be an issue with effective receptive field..

**Comment by windj007** [comment]
AFAIK, masks from PartialConv are pretty narrow (note difference between mask width and size) - so the effective receptive field might be limited due to that. Also consider increasing batch size - we got best results with BS >30 (but larger is better). The issue indeed needs research and is not easy to answer like that :)

**Comment by geekyutao** [comment]
> AFAIK

Thanks for the valuable advice! I'll try it.
